legal/style.css

```
@@ -0,0 +1,53 @@
+body {
+    background-color: #060621;
+	color: #f1f1f1;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+	margin: 0;
+	display: flex;
+	flex-direction: column;
+	min-height: 100vh;
+}
+
+main {
+    padding: 2rem;
```

Add more vertical padding

```suggestion
    padding: 5rem 2rem;
```